### PR TITLE
Increase demo build timeout

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

For the generic build stages it is already set to 30m https://github.com/AudiusProject/audius-client/blob/66afa9a45e5fd0e249ed10c1d572b76a1b4e84c3/.circleci/src/commands/@web-commands.yml#L77

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: .circleci/src/jobs/@web-jobs.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ web-build-demo:
         at: ./
     - run:
         name: build-demo
-        no_output_timeout: 30m
+        git no_output_timeout: 30m
```
Comment (rickyrombo) [inline .circleci/src/jobs/@web-jobs.yml:65]: I'm not sure.I understand this - what does adding `git` here do?

Comment (raymondjacobson) [reply .circleci/src/jobs/@web-jobs.yml:65]: it does not... this was a typo

Comment (rickyrombo) [reply .circleci/src/jobs/@web-jobs.yml:65]: but.. it's the only change?

Comment (raymondjacobson) [reply .circleci/src/jobs/@web-jobs.yml:65]: @sliptype fixed it https://github.com/AudiusProject/audius-client/pull/3322/files

I missed that

**Pull request closed without merging**